### PR TITLE
[README] Corrected some grammatical mistakes

### PR DESCRIPTION
**Pull request by girishponkiya**
Corrected some grammatical mistakes, like added required punctuation marks, corrected singular-plural mistakes, etc.

### CODE REVIEW

**Review by julien-c — APPROVED** [review]

File: docs/source/quickstart.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The library was designed with two strong goals in mind:
 
 - be as easy and fast to use as possible:
 
-  - we strongly limited the number of user-facing abstractions to learn, in fact there are almost no abstractions, just three standard classes required to use each model: configuration, models and tokenizer,
+  - we strongly limited the number of user-facing abstractions to learn, in fact, there are almost no abstractions, just three standard classes required to use each model: configuration, models and tokenizer,
```
Comment (sshleifer) [inline docs/source/quickstart.md:11]:
```suggestion
  - we strongly limited the number of user-facing abstractions to learn. In fact, there are almost no abstractions, just three standard classes required to use each model: configuration, models and tokenizer.
```

File: docs/source/quickstart.md [hunk 2]
```diff
@@ -31,27 +31,27 @@ A few other goals:
 
 ## Main concepts
 
-The library is build around three type of classes for each models:
+The library is build around three types of classes for each model:
 
-- **model classes** which are PyTorch models (`torch.nn.Modules`) of the 8 models architectures currently provided in the library, e.g. `BertModel`
-- **configuration classes** which store all the parameters required to build a model, e.g. `BertConfig`. You don't always need to instantiate these your-self, in particular if you are using a pretrained model without any modification, creating the model will automatically take care of instantiating the configuration (which is part of the model)
-- **tokenizer classes** which store the vocabulary for each model and provide methods for encoding/decoding strings in list of token embeddings indices to be fed to a model, e.g. `BertTokenizer`
+- **model classes** which are PyTorch models (`torch.nn.Modules`) of the 8 models architectures currently provided in the library, e.g., `BertModel`
```
Comment (sshleifer) [inline docs/source/quickstart.md:36]: there are more than 8 architectures I think

File: docs/source/quickstart.md [hunk 3]
```diff
@@ -31,27 +31,27 @@ A few other goals:
...
+- **configuration classes** which store all the parameters required to build a model, e.g., `BertConfig`. You don't always need to instantiate these your-self, in particular, if you are using a pretrained model without any modification, creating the model will automatically take care of instantiating the configuration (which is part of the model)
```
Comment (sshleifer) [inline docs/source/quickstart.md:37]: yourself. In particular,

File: docs/source/quickstart.md [hunk 4]
```diff
@@ -191,7 +191,7 @@ Examples for each model class of each model architecture (Bert, GPT, GPT-2, Tran
 
 #### Using the past
 
-GPT-2 as well as some other models (GPT, XLNet, Transfo-XL, CTRL) make use of a `past` or `mems` attribute which can be used to prevent re-computing the key/value pairs when using sequential decoding. It is useful when generating sequences as a big part of the attention mechanism benefits from previous computations.
+GPT-2, as well as some other models (GPT, XLNet, Transfo-XL, CTRL), make use of a `past` or `mems` attribute which can be used to prevent re-computing the key/value pairs when using sequential decoding. It is useful when generating sequences as a big part of the attention mechanism benefits from previous computations.
```
Comment (sshleifer) [inline docs/source/quickstart.md:194]: This setting can be controlled by the `use_cache` keyword argument.

**Review by sshleifer — APPROVED** [review]
LGTM, thanks!

File: docs/source/quickstart.md (hunk 1 above)
Comment (girishponkiya) [reply docs/source/quickstart.md:11]: All three sub-points (a) start with small caps, and (b) didn't end with a full-stop. So, I thought not to the sentence. Anyway, IMHO, breaking too long sentences is always a good idea.

File: docs/source/quickstart.md (hunk 2 above)
Comment (girishponkiya) [reply docs/source/quickstart.md:36]: I am not familiar with the complete library. So, I (focused on and) have corrected only grammatical mistakes.

Comment (sshleifer) [inline docs/source/quickstart.md:36]:
```suggestion
- **model classes**  e.g., `BertModel` which are 20+ PyTorch models (`torch.nn.Modules`) that work with the pretrained weights provided in the library. In TF2, these are `tf.keras.Model`.

**Comment by sshleifer** [comment]
Thanks for your contribution @girishponkiya , merging!

**Pull request merged**